sunpy/map/mapbase.py

```diff
@@ -1318,7 +1308,7 @@ def _get_mpl_normalizer(self):
 
         Not yet implemented.
         """
-        return None
+        return colors.Normalize(np.nanmin(self.data), np.nanmax(self.data))
```

Comment:
This function can be removed, having the default set in the `plot_settings` dict in the constructor is fine.